store_config_in_the_environment.md

```diff
@@ -4,9 +4,9 @@ Store the configuration in the environment
 General idea
```

Comment:
Generally: Is it an idea to add something about keeping your config seperate even if you don't want to use environment variables? So, this is your preferred approach (as per reasons in the text you quote) but add that at least you should put your config in separate files and add them to .gitignore